functests/4_latency/latency.go

```diff
@@ -428,6 +434,17 @@ func createLatencyTestPod(testPod *corev1.Pod, node *corev1.Node, logName string
 	err = pods.WaitForPhase(testPod, corev1.PodRunning, 2*time.Minute)
 	Expect(err).ToNot(HaveOccurred())
 
+	if runtime, _ := strconv.Atoi(latencyTestRuntime); runtime > 1 {
+		By("Checking actual CPUs number for the running pod")
+		limitsCpusQuantity := testPod.Spec.Containers[0].Resources.Limits.Cpu()
+		RequestsCpusQuantity := testPod.Spec.Containers[0].Resources.Requests.Cpu()
+		//letancy pod is guaranteed
+		//CmpInt64 returns 0 if the quantity is equal to y, -1 if the quantity is less than y, or 1 if the quantity is greater than y
```

Comment:
please remove the comment once we agree about the solution

Reply:
okay will remove it. I just found it handy to illustrate why we expect it to be 0 (as it returns three values). Thank you!